src/pages/landing/ebooks/crypto.js

```diff
@@ -52,7 +75,9 @@ const StocksEbook = () => {
                 ebook_utm_code="cryptocurrencies-ebook"
                 introSub=""
                 introMain={localize('Learn how to trade cryptocurrencies with Deriv')}
-                mainHeaderImage={data['crypto_hero']}
+                mainHeaderImage={
+                    language === 'en' ? data[`crypto_hero`] : data[`crypto_hero_${language}`]
```

Comment:
Not all language have different photos. What will happen for italiano or polish ?